File: Python/1431-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        result = []
        maxCandy = max(candies)
        for i in range(0, len(candies)):
            result.append(candies[i] + extraCandies >= maxCandy)
        return result

File: C++/1431-kids-with-the-greatest-number-of-candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
     vector<bool> result;
     int maxCandy = -1;
     for(int i = 0; i < candies.size(); i++){
       maxCandy = max(maxCandy, candies[i]);
     }
     for(int i = 0; i < candies.size(); i++){
       if(candies[i] + extraCandies >= maxCandy)
         result.push_back(true);
       else
         result.push_back(false);
     }
     return result;
    }
};
